### PR TITLE
Update README to include homebrew installation instructions

### PR DESCRIPTION
**Pull request by dongcarl**
Not too sure what to name the sections... Settled for "Manual"... Any better ideas?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,16 @@ file.
...
 
+### OS X
+
+Install using `brew`:
+
+```
+brew install yank
+```
+
+### Manual
```
Comment (mptre) [inline README.md]: Rename to "From source".

File: README.md [hunk 2]
```diff
@@ -88,6 +88,16 @@ file.
 Installation
 ------------
 
+### OS X
+
+Install using `brew`:
```
Comment (mptre) [inline README.md]: I would remove this paragraph, examples speak louder than words.

File: README.md (hunk 1 above)
Comment (dongcarl) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (dongcarl) [inline README.md]: Done.

**Comment by mptre** [comment]
Thanks! See 9ec39ce.

**Pull request closed without merging**